src/adaptations/device-layer/include/Weave/DeviceLayer/nRF5/BLEManagerImpl.h

```diff
@@ -155,6 +164,16 @@ inline BLEManagerImpl & BLEMgrImpl(void)
     return BLEManagerImpl::sInstance;
 }
 
+inline uint8_t BLEManagerImpl::GetAdvertisingHandle(void)
+{
+    return mAdvHandle;
```

Comment:
Is the handle always initialized to something sane? Is there any way for either the caller or subsequent consumers of the value to know whether the value is sane?

Reply:
It gets initialized here: https://github.com/openweave/openweave-core/blob/5c52bb5b4fe7f3a53e17952af0be4dfe62bf21bb/src/adaptations/device-layer/nRF5/BLEManagerImpl.cpp#L92

This presumes that the Weave stack has been initialized before any call to GetAdvertisingHandle().